src/tactic/command/global_search_trigger.py

```diff
@@ -86,6 +93,39 @@ def execute(my):
         data.update( my.cleanup(caller.get_value("description", no_exception=True) ))
         data.update( my.cleanup(caller.get_value("keywords", no_exception=True) ))
 
+
+        # Updates for adding changed user defined keywords into asset keywords_data column
+        if sobj.column_exists("keywords_data"):
+
+            # On creating new Collections
+            if input.get("update_data").get("_is_collection") and input.get("is_insert"):
+                update_data = input.get("update_data")
+
+                collection_keywords = update_data.get("keywords")
+                collection_name = update_data.get("name")
+
+                keywords_data = sobject.get_json_value("keywords_data", {})
+                keywords_data['user'] = collection_keywords
+                keywords_data['path'] = collection_name
+                sobj.set_json_value("keywords_data", keywords_data)
+                sobj.commit(triggers=False)
+
+            # when user defined keywords column is changed 
+            else:
+                user_keywords = input.get("update_data").get("keywords")
+                if not user_keywords:
+                    user_keywords = ""
```

Comment:
so is it going to reset the keywords all the time since the user may not change the keywords every time they edit the sobject